.gitignore

```diff
@@ -0,0 +1,6 @@
+.idea
```

Comment:
It's also cool to have a gitignore file where we add stuff that shouldn't be versioned.